ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/MacOSX.yml

```diff
@@ -0,0 +1,67 @@
+---
+
+#
+# Updates all packages for macos-based distributions
+#
+
+- name: Check for xcode-tools
+  raw: xcode-select --print-path &> /dev/null
+  register: xcode
+  ignore_errors: yes
+
+- name: Install xcode-tools
```

Comment:
I assume this is here because it's required for the brew installs. In which case can we add a comment to that effect as normally I'd want a compiler installation like this to be in a separate role.

Reply:
no normally xcode-build tools is perfect. it's only on the 10.10 that needs full blown xcode

Reply:
What's in the xcode-tools? And do we have the logic for the full xcode in 10.10 somwhere?

Reply:
We don't have the full logic for xcode, it's essentially a manual step because you have to login to the apple developer account to download the DMG file. build tools gives you all of the command line tools that we need to do builds. xcode is contains the full blown gui application.